book/src/guides/adding_languages.md

```diff
@@ -27,18 +27,19 @@ directory](../configuration.md).
 
 These are the available keys and descriptions for the file.
 
-| Key           | Description                                                   |
-| ----          | -----------                                                   |
-| name          | The name of the language                                      |
-| scope         | A string like `source.js` that identifies the language. Currently, we strive to match the scope names used by popular TextMate grammars and by the Linguist library. Usually `source.<name>` or `text.<name>` in case of markup languages |
-| injection-regex | regex pattern that will be tested against a language name in order to determine whether this language should be used for a potential [language injection][treesitter-language-injection] site. |
-| file-types    | The filetypes of the language, for example `["yml", "yaml"]`  |
-| shebangs      | The interpreters from the shebang line, for example `["sh", "bash"]` |
-| roots         | A set of marker files to look for when trying to find the workspace root. For example `Cargo.lock`, `yarn.lock` |
-| auto-format   | Whether to autoformat this language when saving               |
-| comment-token | The token to use as a comment-token                           |
-| indent        | The indent to use. Has sub keys `tab-width` and `unit`        |
-| config        | Language server configuration                                 |
+| Key                 | Description                                                   |
+| ----                | -----------                                                   |
+| name                | The name of the language                                      |
+| scope               | A string like `source.js` that identifies the language. Currently, we strive to match the scope names used by popular TextMate grammars and by the Linguist library. Usually `source.<name>` or `text.<name>` in case of markup languages |
+| injection-regex     | regex pattern that will be tested against a language name in order to determine whether this language should be used for a potential [language injection][treesitter-language-injection] site. |
+| file-types          | The filetypes of the language, for example `["yml", "yaml"]`  |
+| shebangs            | The interpreters from the shebang line, for example `["sh", "bash"]` |
+| roots               | A set of marker files to look for when trying to find the workspace root. For example `Cargo.lock`, `yarn.lock` |
+| auto-format         | Whether to autoformat this language when saving               |
+| diagnostic_severity | Minimal severity of diagnostic for it to be displayed. (Allowed values: `Error`, `Warning`, `Info`, `Hint`) |
```

Comment:
This is the new entry, the rest is just formatting

Reply:
`diagnostic-severity`, we remap the name to kebab-case via serde

Reply:
sorry, forgot, fixed now